jenkins-slaves/ansible/roles/common/tasks/main.yml

```diff
@@ -21,7 +21,7 @@
           'htop', 'screen', 'lsof', 'bash-completion', 'bzip2', 'rng-tools',
           'ansible', 'zip', 'python2-pip', 'jq', 'libXScrnSaver', 'python3', 'python3-devel',
           'python3-pip', 'glibc-devel', 'zlib-devel', 'podman', 'gcc', 'krb5-devel', 'libstdc++-static', 
-          'java-1.8.0-openjdk-devel', 'java-11-openjdk-devel']
+          'java-1.8.0-openjdk-devel', 'java-11-openjdk-devel', 'sudo']
```

Comment:
Is sudo package really needed?

Reply:
Yeah you are right, it is already there :)